shaikram_abdulguro/activity1/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css">
+    <title>Log In</title>
+</head>
+<style>
+    body{
```

Comment:
Also I don't recommend direct styling,
Pwede ka magcreate separate css and then import it

Reply:
yes sir, ginawa ko lang direct para mabilis ma pasa that time